main/settings.py

```diff
@@ -552,7 +552,7 @@
     "BOOTCAMP_LOG_LEVEL", "INFO", description="The logging level for the application"
 )
 DJANGO_LOG_LEVEL = get_string(
-    "DJANGO_LOG_LEVEL", "DEBUG", description="The log level for Django"
+    "DJANGO_LOG_LEVEL", "INFO", description="The log level for Django"
```

Comment:
Looks fine but I think it can be more configurable with the use of `DEBUG` flag if that is set specifically e.g. in the case of RC and prod the `DEBUG` flag will be false and I think keeping that in mind we can do something like below. What do you think?

```suggestion
    "DJANGO_LOG_LEVEL", "DEBUG" if DEBUG else "INFO", description="The log level for Django"
```

Reply:
I added it to keep it consistent with the other settings `LOG_LEVEL` and `ES_LOG_LEVEL`

Reply:
ok, that makes more sense to follow the `DEBUG`.